v03_pipeline/lib/annotations/shared_test.py

```diff
@@ -42,10 +42,8 @@ def test_sorted_transcript_consequences(self) -> None:
                         gene_id='ENSG00000188976',
                         hgvsc='ENST00000327044.6:c.1667C>T',
                         hgvsp='ENSP00000317992.6:p.Ser556Leu',
-                        lof=None,
-                        lof_filter=None,
-                        lof_flags=None,
-                        lof_info='INTRON_END:881781,EXON_END:881925,EXON_START:881782,DE_NOVO_DONOR_MES:-7.36719797135343,DE_NOVO_DONOR_PROB:0.261170618766552,DE_NOVO_DONOR_POS:-138,INTRON_START:881667,DE_NOVO_DONOR_MES_POS:-138,MUTANT_DONOR_MES:4.93863747168278',
+                        is_lof_nagnag=None,
+                        lof_filters=None,
```

Comment:
could we add a test case for where there are filters?

Reply:
Yep!